tests/test_plotting.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pandas as pd
+# TODO: In the Omigami-core repo the test folder is found inside of the omigami folder and is called test instead of
+#  testS. Should we change that
+from PIL.PngImagePlugin import PngImageFile
+
+from tests.conftest import spectra_match_data_path, spectra_match_data_path_missing_smiles
```

Comment:
you don't need to import the fixtures, if you put this test inside the `tests` directory